.gitignore

```diff
@@ -1 +1 @@
-node_modules
+node_modules
```

Comment:
We do not want "global" repo files changed with a sub-project commits.
The only difference here is that your version of repo root `.gitignore` has no newline character at the end.
Please restore the file with `git checkout <commit-hash> -- ./path/to/file` and commit.
The good commit to take the file from (i.e. commit hash) is `aa6bf18`. Also note the space after `--`, and also the path must be relative to your location within the file system tree.